src/block_mixins/uploadable.js

```diff
@@ -19,6 +19,15 @@ module.exports = {
 
     this.upload_options = Object.assign({}, config.defaults.Block.upload_options, this.upload_options);
     this.inputs.insertAdjacentHTML("beforeend", _.template(this.upload_options.html, this));
+
+    Array.prototype.forEach.call(this.inputs.querySelectorAll('button'), function(button) {
```

Comment:
You can use `Array.from()` here https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from